dashboard/app/models/dsl_defined.rb

```diff
@@ -62,6 +62,7 @@ def self.create_from_level_builder(params, level_params, old_name = nil)
       # Parse data, save updated level data to database
       data, i18n = dsl_class.parse(text, '')
       level_params.delete(:name)
+      level_params.delete(:type) if data[:properties][:type]
```

Comment:
@joshlory It's probably preferable to just always `delete(:type)` but I'm unsure about implications across all of our different level types.  Thoughts?